serverless.js

```diff
@@ -49,39 +48,48 @@ module.exports = class TencentKoa extends Component {
     }
 
     const tencentCloudFunctionOutputs = await tencentCloudFunction(inputs);
-    const apigwParam = {
-      serviceName: inputs.serviceName,
-      description: 'Serverless Framework tencent-koa Component',
-      serviceId: inputs.serviceId,
+
+    const output = {
       region: inputs.region,
-      protocols: apigatewayConf.protocols || ['http'],
-      environment: apigatewayConf.environment || 'release',
-      endpoints: [
-        {
-          path: '/',
-          method: 'ANY',
-          function: {
-            isIntegratedResponse: true,
-            functionName: tencentCloudFunctionOutputs.Name,
-          },
-        },
-      ],
+      functionName: inputs.name,
     };
-    if (apigatewayConf.usagePlan) apigwParam.endpoints[0].usagePlan = apigatewayConf.usagePlan;
-    if (apigatewayConf.auth) apigwParam.endpoints[0].auth = inputs.apigatewayConf.auth;
 
-    this.state.functionName = inputs.name;
-    await this.save();
-    const tencentApiGatewayOutputs = await tencentApiGateway(apigwParam);
+    // if not disable, then create apigateway
+    if (apigatewayConf.disable !== true) {
```

Comment:
When handling booleans in JS, best working way is to simply distinguish between _truthy_ and _falsy_ values.

So cleanest approach would be:

```javascript
if (!apigatewayConf) {
...
```